.github/workflows/gating.yaml

```diff
@@ -129,7 +129,7 @@ jobs:
     - name: Set up Python
       uses: actions/setup-python@v2
       with:
-        python-version: 3.8
+        python-version: 3.9
```

Comment:
If python 3.10 is our target environment, should we use that for the integration tests?

Reply:
But if the tests run successfully on python 3.9, then wouldn't python 3.10 pass as well? For this reason, I thought that the lower bound version would be better since we are testing both 3.9 and 3.10.

Reply:
Exactly, integration tests run only on python 3.9 now. Switch it to python 3.10 please.

Reply:
+1 to use latest greatest!

Reply:
Gotcha, switched to python 3.10